### DISCUSSION TITLE
Kafka exporter

### DISCUSSION THREAD

**Discussion by sgiwaCW** [strimzi/strimzi-kafka-operator#7848] [Ideas]
Please I need help to install and configure kafka exporter in my cluster. I want to hook it to grafana agent.
Please help.

**Comment by scholzj** [comment]
Did you check the docs and the examples? You can just enable it through the `Kafka` custom resource. There are also some examples for configuring it with Prometheus operator, but that depends on you how you install and manage Prometheus.

**Reply by sgiwaCW** [reply]
Thanks for the quick response.
I want to try to deploy the kafka exporter with this yaml file

apiVersion: kafka.strimzi.io/v1beta1
kind: Kafka
metadata:
  name: my-cluster
spec:
  # ...
  kafkaExporter:
    image: my-org/my-image:latest (1)
    groupRegex: ".*" (2)
    topicRegex: ".*"

my only concern is if v1beta1 is the updated version or v1.
Do you think this will work?

**Reply by scholzj** [reply]
Sorry, I don't follow what `v1beta1` has to do with this.

**Reply by sgiwaCW** [reply]
this is the error that I am getting, 

E1222 15:51:07.151833   96368 memcache.go:255] couldn't get resource list for external.metrics.k8s.io/v1beta1: Got empty response for: external.metrics.k8s.io/v1beta1

**Reply by scholzj** [reply]
Sorry, but where do you get it? This IHMO doesn't have anything to so with Strimzi.
